ansible/istio/defaults/main.yml

```diff
@@ -2,6 +2,9 @@
 # If set to true, the minishift role is first executed to setup a proper VM for installing Istio locally
 run_minishift_role: false
 
+# Whether the cluster is an Openshift (oc) or stock Kubernetes (k8s) cluster
+cluster_flavour: oc #k8s
```

Comment:
We should use `ocp` instead of `oc` which is the acronym of the binary go client and also because we use for kubernetes a 3letters acronym too

Reply:
Makes sense! I'll update it immediately